sdk/python/feast/cli.py

```diff
@@ -155,13 +155,23 @@ def feature_set_create(filename):
 
 
 @feature_set.command("describe")
-@click.argument("name", type=click.STRING)
-def feature_set_describe(name: str):
+@click.option(
+    "--name", "-n", help="Name of feature set", required=True, type=click.STRING
```

Comment:
I think a positional argument for the name makes a bit more sense here. Did you have a reason for making this change?

Reply:
Not exactly other than offering the `help` option to explain what name refers to. (though I guess it's self-explanatory)